components/eventing-controller/pkg/backend/eventmesh/eventmesh.go

```diff
@@ -85,11 +85,11 @@ func (em *EventMesh) Initialize(cfg env.Config) error {
 
 // getWebHookAuth returns the webhook auth config from the given env config
 // or returns an error if the env config contains invalid grant type or auth type.
-func getWebHookAuth(cfg env.Config, credentials *OAuth2ClientCredentials) *types.WebhookAuth {
+func getWebHookAuth(credentials *OAuth2ClientCredentials) *types.WebhookAuth {
 	return &types.WebhookAuth{
 		ClientID:     credentials.ClientID,
 		ClientSecret: credentials.ClientSecret,
-		TokenURL:     cfg.WebhookTokenEndpoint,
+		TokenURL:     credentials.TokenURL,
```

Comment:
hope `credentials.TokenURL` is set for both cases (feature flag enabled & disabled)

Reply:
It is set here https://github.com/kyma-project/kyma/blob/88da789a122950f526ea5243e813bcfb5320f266/components/eventing-controller/controllers/backend/reconciler.go#L732.